### PR TITLE
Add SignUpTest

### LINKED ISSUE #1: Please add test script for verifying elements on signup page
Please add test script for verifying elements on signup page (http://staging.aceinvoice.com/sign_up)

### PR DESCRIPTION
**Pull request by farhanacherada**
Fixes #1 

@prabhakarbattula _a please review.

### CODE REVIEW

**Comment by prabhakarbattula** [comment]
@farhanacherada 

The test added in this PR was for signing up a new user.  The issue #1 pertains to verifying the various elements on signup page.  

Please send a separate PR for just verifying elements on signup page.

This PR is good for signing up a new user. 

Please add .classpath and .project and .settings to gitignore.
Further please remove those files from this PR.

**Comment by farhanacherada** [comment]
@prabhakarbattula Can you please show me an example for verifying elements?

**Comment by prabhakarbattula** [comment]
@farhanacherada 
image

File: AceInvoice Test/src/aceinvoice/SignUpTest.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+public class SignUpTest {
+	
+	public static void main(String[] args) {
+	
+    WebDriver driver=new ChromeDriver();
+	 
+     driver.get("http://staging.aceinvoice.com/sign_up"); 
+	
+     //Page 1
+     driver.findElement(By.name("email")).sendKeys("test+" + Math.random() + "@example.com");
```
Comment (prabhakarbattula) [inline AceInvoice Test/src/aceinvoice/SignUpTest.java]: Please create a random method in common files folder and can make use of it in all test cases.  Example provided.
https://github.com/prabhakarbattula/aceinvoice-selenium-shweta/blob/master/src/common_files/Randomnumber.java

File: AceInvoice Test/src/aceinvoice/SignUpTest.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+     driver.findElement(By.name("get_started")).click();
+     driver.findElement(By.name("password")).sendKeys("welcome");
+     driver.findElement(By.name("password_confirmation")).sendKeys("welcome");
+     driver.findElement(By.name("continue")).click();
+ 
+     driver.manage().timeouts().implicitlyWait(5, TimeUnit.SECONDS);
+
+	 
+     //Page 2
+     driver.findElement(By.name("user[first_name]")).sendKeys("sample");
```
Comment (prabhakarbattula) [inline AceInvoice Test/src/aceinvoice/SignUpTest.java]: Add random to firstname like sample+random (sample+123232)
Add random to lastname also (Test+123232)

File: AceInvoice Test/src/aceinvoice/SignUpTest.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+     Select dropDownTimezone= new Select(driver.findElement(By.name("user[time_zone]")));
+     dropDownTimezone.selectByVisibleText("(GMT+05:30) Kolkata");
+	 
+     Select dropDownDateformat= new Select(driver.findElement(By.name("user[date_format]")));
+     dropDownDateformat.selectByVisibleText("dd/mm/yyyy");
+	 
+     Select dropDownStartDate= new Select(driver.findElement(By.name("user[start_of_week]")));
+     dropDownStartDate.selectByVisibleText("monday");
+     
+     driver.findElement(By.name("commit")).click();
```
Comment (prabhakarbattula) [inline AceInvoice Test/src/aceinvoice/SignUpTest.java]: Please verify the firstname and lastname which was provided earlier or profile name as displayed on landing page.

**Comment by prabhakarbattula** [comment]
@farhanacherada 

Please update as per feedback.

**Comment by farhanacherada** [comment]
@prabhakarbattula _a Updated the PR by creating the random method in common files folder. 

>Please verify the firstname and lastname which was provided earlier or profile name as displayed on 
  landing page.

Landing page didn't show the First name and last name - http://take.ms/mbcuF.

**Comment by prabhakarbattula** [comment]
@farhanacherada 

The scenario is explained in the video https://www.useloom.com/share/20e8edfa9bac4695832f1617c7e443e3

Please update test case accordingly.

**Comment by farhanacherada** [comment]
@prabhakarbattula _a Updated the PR. Please review.

**Comment by farhanacherada** [comment]
@prabhakarbattula _a Updated the changes. Please review.

File: AceInvoiceTests/src/testcases/SignUpTest.java [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+    public void before() {
+       driver.get("http://staging.aceinvoice.com/sign_up");
+    }
+
+    @Test
+    public void verifySignUpPageElements() {
+
+      RandomNumber rn = new RandomNumber();
+      int randomNumber = rn.random();
+
```
Comment (prabhakarbattula) [inline AceInvoiceTests/src/testcases/SignUpTest.java]: Use variables as under:

string email = randomNumber + "@example.com";
string firstName="first-name" + randomNumber;
string lastName="last-name" + randomNumber;
string orgName="org-name" + randomNumber;

Use them in lines 30, 38, 39, 53, 54, 64, 67 & 70

**Comment by prabhakarbattula** [comment]
@farhanacherada 

Please update.

**Comment by farhanacherada** [comment]
@prabhakarbattula _a Updated the PR. Please review.

**Pull request merged**